### DISCUSSION TITLE
Problem of Writing Solution with Iteration Number

### DISCUSSION THREAD

**Discussion by ardao13** [su2code/SU2#2127] [General]
I use the WRT_VOLUME_OVERWRITE = NO option with the aim of printing the volume file with the iteration number added to the end of the filename at specified intervals. Although it says it's writing on the screen, I can't see these files in the working folder. I initially encountered this issue in my own case, then tried it in the quickstart case and on different computers, and still encountered the same problem.

![Screenshot from 2023-09-08 11-31-54](https://github.com/su2code/SU2/assets/73038236/34d8da34-1967-4326-85e0-2361480f3efa)

![Screenshot from 2023-09-08 11-33-09](https://github.com/su2code/SU2/assets/73038236/16cbcabe-a6c7-4a1a-85ad-1252d8c93139)

**Comment by bigfooted** [comment]
Strange, it works for me. Is this on linux? Is it with the latest version (8.0 "Harrier") ?

**Reply by ardao13** [reply]
Hi @bigfooted,

This issue goes away with latest version (8.0 "Harrier"). #2036 

It exists at version 751 therefore I will go on with latest version :)
